rclpy/rclpy/qos_event.py

```diff
@@ -72,6 +74,18 @@ class QoSSubscriptionEventType(IntEnum):
         ('not_alive_count_change', 'int'),
     ])
 
+"""
+Payload type for Subscription Incompatible QoS callback.
+
+Mirrors rmw_requested_incompatible_qos_status_t from rmw/types.h
+"""
+QoSRequestedIncompatibleQoSInfo = NamedTuple(
+    'QoSRequestedIncompatibleQoSInfo', [
+        ('total_count', 'int'),
+        ('total_count_change', 'int'),
+        ('last_policy_id', 'int'),
```

Comment:
An int is not really clear for this.
Maybe a python enum https://docs.python.org/3/library/enum.html?

Reply:
👍 for an enum for the policy id. Also, as I mentioned in the rmw pr, I think `policy_kind` makes more sense. The fact that it is an enum or is backed with an int versus a string versus a type or whatever is an implementation detail.

Reply:
I've introduced a `QoSPolicyKind` `IntEnum`, but I'll figure out how to specify that in a `NamedTuple` later.